### ISSUE TITLE
Support for CoreOS

### ISSUE THREAD

**Issue by kerneltime** [vmware/docker-volume-vsphere#730]
This involves 

- [ ] publishing the plugin as docker images

- [ ] document and/or automate as far as possible the deployment of plugin on core os.

**Comment by pdhamdhere** [comment]
Did following to use vSphere Docker Volume service on CoreOS.

- Build vSphere Docker Volume Service container from latest release (Thanks to @kerneltime )
```
cat Dockerfile 
FROM ubuntu:16.04

RUN apt-get -y update && apt-get -y install rpm2cpio cpio 
ADD https://github.com/vmware/docker-volume-vsphere/releases/download/0.8.1/docker-volume-vsphere-0.8.1-1.x86_64.rpm /
RUN rpm2cpio docker-volume-vsphere-0.8.1-1.x86_64.rpm | cpio -idmv
CMD /usr/local/bin/docker-volume-vsphere -port 1019 -log_level=debug
```
```
docker build -t dvs -f Dockerfile .
```
- Configure Docker to use shared mount: Create `/etc/systemd/system/docker.service.d/slave-mount-flags.conf` and add following
```
[Service]
MountFlags=shared
```
- `systemctl daemon-reload`
- `systemctl restart docker`

- Launch vSphere Docker Volume Service Container
```
docker rm -f dvs; sudo docker run --privileged -d -v /var/run/docker/plugins/:/var/run/docker/plugins/ -v /run/docker/plugins/:/run/docker/plugins/ -v /var/run/docker.sock:/var/run/docker.sock -v /dev:/dev -v /mnt:/mnt:shared --name dvs dvs
```

**Comment by kerneltime** [comment]
Relevant: https://github.com/docker/docker/issues/20656

**Comment by pdhamdhere** [comment]
Will be fixed with #886

**Comment by shuklanirdesh82** [comment]
closing as per https://github.com/vmware/docker-volume-vsphere/issues/730#issuecomment-284587136
